modules/system/holoportos/udiskie.nix

```
@@ -0,0 +1,40 @@
+{ config, pkgs, ... }:
+let
+  wrapSystemdScript = (
+    program: ''
+      source ${config.system.build.setEnvironment}
+      exec ${program}
+    ''
+  );
+in rec {
+  environment.systemPackages = with pkgs; [
+    udiskie
+    udisks2
+  ];
+  services = {
+    udisks2.enable  = true;
+  };
+  systemd.user.services = {
```

Having as few indentation levels as possible is more readable:

```suggestion
  systemd.user.services.udiskie = {
```